src/pages/operations/OperationList.tsx

```diff
@@ -166,7 +158,7 @@ const OperationList: FC = () => {
               image={<Icon name="status" className="empty-state-icon" />}
               title="No operations found"
             >
-              <p>There are no ongoing operations in this project.</p>
+              <p>There are no ongoing operations right now.</p>
```

Comment:
Maybe "right now" is already implicit in "ongoing"?
```suggestion
              <p>There are no ongoing operations.</p>
```

Reply:
Yeah you're right "ongoing" already implies that.